docs/src/index.md

```diff
@@ -2,12 +2,15 @@
 *Statically sized arrays for Julia*
 
 **StaticArrays** provides a framework for implementing statically sized arrays
-in Julia (≥ 0.5), using the abstract type `StaticArray{Size,T,N} <: AbstractArray{T,N}`.
+in Julia (≥ v1.6)[^1], using the abstract type `StaticArray{Size,T,N} <: AbstractArray{T,N}`.
```

Comment:
I think we can just drop this reference to version.  At this point, there's a StaticArrays release for every version of Julia that people are likely to be using, and the old releases will continue to work for older Julia versions.

Reply:
Oki, even better - I just removed references to versions now then.